product-scenarios/1-integrating-systems-that-communicate-in-heterogeneous-message-formats/1.1-converting-soap-to-json/scenario_1.1-test-artifacts/scenario_1_1-synapse-config/src/main/synapse-config/api/1_1_1_soap_to_json_using_payload_factory.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<api context="/1_1_1_soap_to_json_using_payload_factory" name="1_1_1_soap_to_json_using_payload_factory"
+     xmlns="http://ws.apache.org/ns/synapse">
+    <resource methods="POST">
+        <inSequence>
+            <payloadFactory description="Build Response Payload in JSON format" media-type="json">
+                <format>{ "LookupCity": {"Zip": $1}}</format>
+                <args>
+                    <arg xmlns:tem="http://tempuri.org" evaluator="xml" expression="//tem:LookupCity/tem:Zip"/>
+                </args>
+            </payloadFactory>
+            <send description="Send Request Payload to SOAP endpoint">
+                <endpoint key="EP_JSONEndpoint"/>
+            </send>
+        </inSequence>
+        <outSequence>
+            <payloadFactory description="Build SOAP request payload" media-type="xml">
+                <format>
+                    <soapenv:Envelope xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/">
+                        <soapenv:Body>
+                            <LookupCityResult xmlns="http://tempuri.org">
+                                <LookupCity>
+                                    <City>$1</City>
+                                    <State>$2</State>
+                                    <Zip>$3</Zip>
+                                </LookupCity>
+                            </LookupCityResult>
+                        </soapenv:Body>
+                    </soapenv:Envelope>
+                </format>
+                <args>
+                    <arg evaluator="json" expression="$.LookupCityResult.City"/>
+                    <arg evaluator="json" expression="$.LookupCityResult.State"/>
+                    <arg evaluator="json" expression="$.LookupCityResult.Zip"/>
+                </args>
+            </payloadFactory>
+            <respond description="Respond to client"/>
+        </outSequence>
+        <faultSequence>
+            <makefault version="soap11">
+                <code xmlns:soap11Env="http://schemas.xmlsoap.org/soap/envelope/"
+                      expression="get-property('ERROR_CODE')"/>
+                <reason expression="get-property('ERROR_MESSAGE')"/>
+            </makefault>
+            <header action="remove" name="To" scope="default"/>
```

Comment:
We do not need to remove To property specifically since we use to respond mediator to send the client it will remove this internally.